samples/cpp/benchmark_app/main.cpp

```diff
@@ -437,7 +437,7 @@ int main(int argc, char* argv[]) {
                             std::stringstream strm(it_device_nstreams->second);
                             std::map<std::string, std::string> devices_property;
                             ov::util::Read<std::map<std::string, std::string>>{}(strm, devices_property);
-                            for (auto it : devices_property) {
+                            for (auto& it : devices_property) {
```

Comment:
```suggestion
                            for (const auto& it : devices_property) {
```
You don't change `it`